docs/docs/addons-perf.md

```diff
@@ -14,6 +14,10 @@ next: test-utils.html
 import Perf from 'react-addons-perf' // ES6
 var Perf = require('react-addons-perf') // ES5 with npm
 var Perf = React.addons.Perf; // ES5 with react-with-addons.js
+
+//If Perf is not accessible from console i.e throws RefrenceError: 
+//Perf is not defined, create a global 
+window.Perf = Perf;
```

Comment:
It has been mentioned in the solution of issue #6174 . So I think it should be included so developers know how to solve it when they run into reference error.